package.json

```diff
@@ -19,7 +19,7 @@
     "prepare": "git config core.hooksPath .git/hooks/ && simple-git-hooks"
   },
   "simple-git-hooks": {
-    "pre-commit": "pnpm lint-staged"
+    "pre-commit": "./node_modules/.bin/nano-staged"
```

Comment:
is this faster? or why do we need this path

Reply:
Yes, package manager start time adds more to commit time